scripts/exgdas_global_marine_analysis_ecen.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
These scripts should be in global-workflow if they are to follow what is being done for atmosphere, snow, and land.

Reply:
We will at some point @aerorahul , but the first few iterations will be merged in the gdasapp.

Reply:
Ok.